internal/faustjs.org/docs/migrationPath/overview.mdx

```diff
@@ -2,6 +2,279 @@
 slug: /migrationPath/overview
 title: Migration from Legacy Faust
 ---
-Migrate from old version of Faust.js to new Faust
 
-### This document is in progress. Check back soon for content!
\ No newline at end of file
+## General Guidelines
+
+Currently, there is no great automated way to migrate from previous versions of Faust that use GQty to the recent version using Apollo. This is primarily due to their striking differences in the way the queries are performed.
```

Comment:
I would make sure the language stays very positive. Maybe change things like "no great automated way" to something like "Currently migration from previous versions of Faust that use GQty is a manual process. To make the process easier we have outlined the necessary steps below."